src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -421,6 +421,81 @@ public async Task VerifyBankAsync(Guid organizationId, int amount1, int amount2)
         }
     }
 
+    public async Task<(Organization organization, OrganizationUser organizationUser, Collection defaultCollection)> SignupClientAsync(OrganizationSignup signup)
```

Comment:
This is a newly added method responsible for signing up a client organization for an MSP as part of Consolidated Billing.

Reply:
Non-blocking comment: Hmmm. Yet another fork on creating an organization. @eliykat Any thoughts on how we mitigate/improve this fragmentation?

Reply:
@vincentsalucci @eliykat Just to emphasize, this is relatively peculiar situation given we're creating organizations with paid features that do not have any payment information. That's all managed at the provider level so I need to be able to skip that process.

Reply:
I have a tech debt task to move org signup methods into their own command: https://bitwarden.atlassian.net/browse/AC-1517

My vision here is a `CreateOrganizationCommand` with different public methods for different create flows (via web app, via provider portal, via Bitwarden portal, etc...)

I don't like adding to `OrganizationService` but I think it may as well live here with the other signup methods so that they're all together for us to move later. So I'm OK with this PR, we just gotta continue breaking up OrganizationService (AC Team work).